hbase-server/src/test/java/org/apache/hadoop/hbase/client/HConnectionTestingUtility.java

```diff
@@ -39,12 +39,17 @@
    * probably not what you want.
    * @param conf configuration
    * @return ConnectionImplementation object for <code>conf</code>
-   * @throws ZooKeeperConnectionException
+   * @throws IOException
```

Comment:
Empty javadoc like this provokes complaint by checkstyle. Just remove it.

Reply:
Will do.